recipe/build.sh

```diff
@@ -45,6 +45,7 @@ LINKFLAGS="${LINKFLAGS} -L${LIBRARY_PATH}"
     cxxflags="${CXXFLAGS}" \
     linkflags="${LINKFLAGS}" \
     --layout=system \
+    --with-python \
```

Comment:
@jschueller, when this is given only python lib is built. You can see that in the logs.